pkg/bootkube/create.go

```diff
@@ -164,7 +164,7 @@ func (c *creater) createManifests(manifests []manifest) (ok bool) {
 	create := func(m manifest) {
 		if err := c.create(m); err != nil {
 			ok = false
-			UserOutput("Failed creating %s: %v", m, err)
+			UserOutput("Failed creating %s: %v\n", m, err)
```

Comment:
Since everywhere that calls `UserOutput` ends with a newline, why don't we have the function include it instead?

Reply:
In some cases, two newlines are passed for display purposes, and knowing the function adds newlines does not conform to the fmt.Printf API (Printf does not add newlines by default). Being explicit on the output formatting is generally better, I think.

Reply:
I'd be happy with replacing this with fmt.Printf (so we'd get vet warnings) or a logger struct that we passed. UserOutput feels a bit unnecessary.